### DISCUSSION TITLE
s/suggestion-for-open-source-jumpserver-oneterm

### DISCUSSION THREAD

**Discussion by giscus** [ops-coffee/blog#90] [Announcements]
# s/suggestion-for-open-source-jumpserver-oneterm

None

https://blog.ops-coffee.cn/s/suggestion-for-open-source-jumpserver-oneterm.html

**Comment by ttkanni** [comment]
试过很多款堡垒机，OneTerm虽然简约，但功能也少（基础使用足够），推荐试试Jumpserver~
